SEED-platform/seed
# Update verbiage relating to recent matching changes

#### Any background context you want to provide?

#### What's this PR do?
- On file import, remove stale statement about matches not being found
- On column settings page, update text detailing the possible ways to change columns.
- For both of the above changes, translate text.

#### How should this be manually tested?
- Review text on Org Column Settings page
- Successfully import a file and see that the **"SEED could not locate any existing matches"** text has been removed.

#### What are the relevant tickets?
#1571 

#### Screenshots (if appropriate)
